docs/rules/max-statements-per-line.md

```diff
@@ -0,0 +1,152 @@
+# Limit Maximum Number of Statements Per Line (max-statements-per-line)
+
+A line of code containing too many statements can be difficult to read. Code is generally read from the top down, especially when scanning, so limiting the number of statements allowed on a single line can be very beneficial for readability and maintainability.
+
+```js
+function () { var bar; if (condition) { bar = 1; } else { bar = 2; } return true; } // too many statements
+```
+
+## Rule Details
+
+This rule allows you to configure the maximum number of statements allowed per line.  The default is 1.
+
+Examples of **incorrect** code for a maximum of 1:
+
+```js
+/*eslint max-statements-per-line: [2, 1]*/
```

Comment:
We don't do rules in this format anymore because it's unckear what 1 means. I'd like to see the rule be configured like this:

```
max-statements-per-line: ["error ", {
    max: 1
}]
```

That's more the way we've been doing rules lately.